components/_preview.hbs

```diff
@@ -14,6 +14,11 @@
     <!--[if gt IE 8]><!-->
       <script>document.documentElement.className = document.documentElement.className.replace('no-js','has-js')</script>
     <!--<![endif]-->
+    <script>
+      window.__EQ_SESSION_TIMEOUT__ = 180;
```

Comment:
This `__EQ_SESSION_TIMEOUT__` pair will require changing too if we remove eQ from the name (see previous comment)